core/src/test/java/org/elasticsearch/search/indicesboost/SimpleIndicesBoostSearchIT.java

```diff
@@ -77,31 +75,154 @@ public void testIndicesBoost() throws Exception {
         logger.info("Hit[1] {} Explanation {}", response.getHits().getAt(1).index(), response.getHits().getAt(1).explanation());
         assertThat(response.getHits().getAt(0).index(), equalTo("test2"));
         assertThat(response.getHits().getAt(1).index(), equalTo("test1"));
+    }
 
-        logger.info("--- DFS_QUERY_THEN_FETCH");
+    public void testIndicesBoostWithAlias() throws Exception {
+        createIndex("alias_test1", "alias_test2");
```

Comment:
maybe call the concrete indices "index1" and "index2", otherwise one may think they are aliases :)